packages/flutter_tools/lib/src/desktop_device.dart

```diff
@@ -137,6 +138,7 @@ abstract class DesktopDevice extends Device {
       <String>[
         executable,
       ],
+      environment: _computeEnvironment(debuggingOptions, traceStartup, route),
```

Comment:
Do these options work in release mode on other platforms? Since they won't currently on desktop, maybe we should see if this is non-empty and if the build mode is release, output a warning that the options will be ignored and to file an issue describing their use case.

Reply:
The tooling will already ignore the unsupported values when constructing the debugging options object:

https://github.com/flutter/flutter/blob/90bc7c64dd3c02922e4bad49f6001fe32b86ae54/packages/flutter_tools/lib/src/commands/run.dart#L374

That seems like a reasonable place to insert some logging to warn users if they are passing flags that have no effect.

Reply:
Filled https://github.com/flutter/flutter/issues/67315